scripts/postprocess.py

```diff
@@ -190,9 +202,12 @@ def _calc_metrics(cols, df):
 #     "nodes":
 # }
 def _parse_line_based(fout, bench_name, nodes):
-    if "dlaf" in bench_name:
+    if bench_name.startswith("chol"):
```

Comment:
Note to self: this should be reverted. @albestro this should answer your question in https://github.com/eth-cscs/DLA-Future/pull/443#pullrequestreview-848713690.

I initially separated the parsing for the different miniapps, but didn't revert the condition completely after making the parsing more generic again.